Hazel/src/Platform/Windows/WindowsWindow.cpp

```diff
@@ -172,5 +178,46 @@ namespace Hazel {
 	{
 		return m_Data.VSync;
 	}
+	
+	void WindowsWindow::SetWindowMode(const WindowMode& mode)
+	{
+		HZ_ASSERT(m_Window, "Failed to retrieve window.");
+		if (mode == m_Data.Mode) return;
+
+		unsigned int width = 0, height = 0;
+
+		m_PrimaryMonitor = glfwGetPrimaryMonitor();
+		m_BaseVideoMode = *(glfwGetVideoMode(m_PrimaryMonitor));
+
+		GLFWmonitor* monitor = nullptr;
+
+		if (mode == WindowMode::Windowed) {
```

Comment:
I would suggest changing this to a switch case statement for readability, since that's what we usualy do in the code base for enums, including the `HZ_CORE_ERROR` when an invalid mode has been passed (default case)